### DISCUSSION TITLE
Step 4, Add Copilot suggestion from comment: Some jobs were not successful not sure why

### DISCUSSION THREAD

**Discussion by Peace-Aielumoh** [skills/.github#290] [Code with Copilot]
![Screenshot (180)](https://github.com/skills/.github/assets/77479372/f3e2af50-765e-4419-a6fb-3a6088a81369)

**Comment by dmitryofitserov** [comment]
Try to see more informative message regarding a failure under the GitHub Actions tab of your repository

**Reply by sinsukehlab** [reply]
Yes, you can know why the jobs have failed by reading the error message under the GitHub Actions tab.
https://github.com/Peace-Aielumoh/skills-copilot-codespaces-vscode/actions/workflows/4-copilot-comment.yml

You can then see the error code 204. You can see the details of the workflow and get to the cause.

Cf. #286

**Reply by Peace-Aielumoh** [reply]
Thank you
